Src/CSharpier.Tests/FormattingTests/TestFiles/VariableDeclarations.cst

```diff
@@ -76,8 +76,20 @@ class ClassName
                 + SqlParameterCollectionExtensions.DefaultValueColumnWidth
         ];
 
-        var someLongValue_________________ =
-            memberAccessExpression[elementAccessExpression].theMember______________________________;
+        var someLongValue_________________ = memberAccessExpression[
+            elementAccessExpression
```

Comment:
Not sure if this is a regression as part of this change, or just exposed by this test case update:
Can you make it prefer breaking at the `.` than `[`?

Seeing method chains across multiple lines is pretty normal, whereas breaking on array access looks strange and should be used as a last resort.